README.rst

```diff
@@ -51,7 +51,10 @@ It basically means:
 - generate metadata for haskell repo after main
   tree sync is done, using N+1 cores
 
-Build issues? If you see packages from the gentoo repository being
+Build issues?
+=============
+
+If you see packages from the gentoo repository being
 selected over the haskell overlay packages and failing to build,
 check the priorities in the configuration files under
 ``/etc/portage/repos.conf/``. The default priority for the gentoo
```

Comment:
Also, it would be good if you cited the Gentoo docs or wiki that give more detail on how overlay priority works.

Reply:
Good point - will do!